pos_payment_method_cashdro/static/src/scss/pos_cashdro_terminal.scss

```
@@ -0,0 +1,7 @@
+button.cashdro-transaction-start,  button.cashdro-transaction-cancel {
+    /*Same css values as ".screen .top-content .button" */
+    font-size: 20px;
+    padding: 3px 13px;
+    line-height: 32px;
+    border-radius: 3px;
+}
```

```suggestion
.pos {
    button.cashdro-transaction-start,  button.cashdro-transaction-cancel {
        /*Same css values as ".screen .top-content .button" */
        font-size: 20px;
        padding: 3px 13px;
        line-height: 32px;
        border-radius: 3px;
    }
}
```

It'd be even better to make it more restrictive, but I don't know the widget's css classes by heart